packages/viewer/src/scenes/renderResult.ts

```diff
@@ -0,0 +1,40 @@
+import { Frame } from '../types';
+import { StreamApi } from '@vertexvis/stream-api';
+import { Result } from '../stream/result';
+import { StreamApiEventDispatcher } from '../stream/dispatcher';
+
+export interface RenderResultIds {
+  animationId?: string;
+  correlationId?: string;
+}
+
+export class RenderResult implements Result {
+  public data = undefined;
+  public onAnimationCompleted: StreamApiEventDispatcher<string | undefined>;
+  public onFrameReceived: StreamApiEventDispatcher<Frame.Frame | undefined>;
```

Comment:
It'd be better if this was a type `StreamApiEventDispatcher<Frame.Frame>`. This way the caller doesn't have to do a null check.